app/api/root_api.rb

```diff
@@ -7,7 +7,15 @@ class RootApi < Grape::API
     error!('Not Found', 404)
   end
 
-  # mount NotificationsApi
+  rescue_from Grape::Exceptions::ValidationErrors do |e|
+    rack_response({
+      status: e.status,
+      error_msg: e.message
+    }.to_json, 400)
+  end
+
+  mount Login
+  mount Register
```

Comment:
We might rename these classes to `LoginApi` and `RegisterApi` to describe better their purpose, wdyt? In their current form, they are more similar to a service object class from my humble point of view.